app/scripts/controllers/audits.js

```diff
@@ -407,12 +411,20 @@ angular.module('openhimConsoleApp')
       $scope.settings.filter.dateEnd = '';
       $scope.settings.list.tabview = 'same';
 
-      // clear filters by loading audits page again with no filters
-      var baseUrl = $location.protocol() + '://' + $location.host() + ':' + $location.port() + '/#/';
-      var path = 'audits';
-      window.location = baseUrl + path;
+      // get the filter params object before clearing them
+      var filterParamsBeforeClear = JSON.stringify( angular.copy( $location.search() ) );
 
-      $scope.refreshAuditsList();
+      // clear all filter parameters
+      $location.search({});
+
+      // get the filter params object after clearing them
+      var filterParamsAfterClear = JSON.stringify( angular.copy( $location.search() ) );
+
+      // if the filters object stays the same then call refresh function
+      // if filters object not the same then angular changes route and loads controller ( refresh )
+      if ( filterParamsBeforeClear === filterParamsAfterClear ){
```

Comment:
I think you can skip converting these to string and rather use `angular.equals` to do this check directly on the objects.